### ISSUE TITLE
how to have artillery.io integration with CI

### ISSUE THREAD

**Issue by sreedevi-1991** [artilleryio/artillery#797]
I need artillery.io integration with CI and check if  it is possible to integrate artillery performance scripts before the script is deployed to production. Can we like stop/break the deployment if the request breaks?

**Comment by hassy** [comment]
Yes - the most common way is to check some condition based on Artillery's JSON log with a shell script and fail a pipeline step if it's not met.

**Comment by tomashyam** [comment]
can you post an example of your script ?
